doc/UsersGuide/source/packagemanager.rst

```diff
@@ -1,70 +1,143 @@
+number and priority is installed.
+
+Package Management in OMEdit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TBD: Show how to install new packages and manage installed ones
+
+Running Conversion Scripts in OMEdit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TBD: Show how to run conversion scripts in OMEdit
+
+Automatically Loaded Packages in OMEdit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When you start OMEdit, some packages are automatically loaded into the environment, and shown in the Libraries
+Browser. You can configure which ones are loaded from the Tools|Options|Libraries menu.
+
+Please note that automatically loaded libraries may be in conflict with the dependencies of packages that you may
+then load from the File menu. For example, if you automatically load MSL ``4.0.0``, and then open a library that
+uses MSL ``3.2.3``, you get a conflict, because the former is not backwards-compatible with the latter. In this
+case you have three options: 
+
+- cancel the operation;
+- unload the conflicting library, i.e., MSL ``4.0.0``, and load the most recent one compatible with the one
+  declared in the uses annotation of the opened library, i.e., MSL ``3.2.3``;
+- upgrade the library you just opened to use the already loaded one, i.e. MSL ``4.0.0``, by running the automatic
+  conversion script; note that this operation is irreversible and must be carefully planned, considering all the
+  users of the library that is undergoing automatic conversion.
+
+Manually Loading Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to maintain full control over which libraries are opened, you can use the File | Open Model/Library Files(s)
+menu command in OMEdit to open the libraries one by one from specific locations in your file system. Note,
+however, that whenever a library is loaded, its dependencies as declared in the uses annotation will automatically
+be loaded. If you want to avoid that, you need to load the library dependencies in reverse order, so that the
+intended library dependencies are already loaded when you open the library that needs them.
+
+If you are using the Interactive Environment, you can use the ``loadFile()`` command to load libraries from
+specific locations on the file system, also in reverse dependency order, unless you also set the optional
+``uses = false`` input argument to disable the automatic loading of dependencies.
+
+Using the Package Manager from the Interactive Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Package Manager can also be used from the Interactive Environment command line shell. Here is a list
+of examples of relevant commands; please type them followed by ``getErrorString()``, 
+e.g., ``updatePackageIndex();getErrorString()``, in order to get additional information,
+notifications and error messages.
+
+- ``updatePackageIndex()``: this command puts the Package Manager in contact with the OSMC servers and updates
```

Comment:
I would also cross-reference the corresponding entries in the API documentation

Reply:
Will do that, I need to figure out how to add cross references to verbatim text

Reply:
Unfortunately, I have no idea how to do that. The API function documentation is in interface.inc, which is automatically generated but I cannot see. @sjoelund, @adeas31 can you at least put in one example with information where should I look to find the references?